eth-contracts/scripts/migrate-contracts.js

```diff
@@ -114,7 +114,7 @@ module.exports = async callback => {
     console.log("Content service folder doesn't exist, probably running via E2E setup scripts", e)
   }
 
-  const dappOutput = path.join(os.homedir(), './audius')
+  const dappOutput = path.join(os.homedir(), '/.audius')
```

Comment:
broke in this commit: https://github.com/AudiusProject/audius-protocol/commit/ed20eb259bf9d7d6c21fa95c9eca959ebe33dc54#diff-a574731c4a9e309c64d6bba06ff2b474R117

Reply:
whoops 😅thanks for the catch!